Assets/HoloToolkit/Sharing/Tests/Scripts/ImportExportAnchorManager.cs

```diff
@@ -701,13 +693,14 @@ private void AnchorStoreReady(WorldAnchorStore store)
         /// </summary>
         private void CreateAnchorLocally()
         {
-            if (thisAnchor.isLocated)
+            WorldAnchor anchor = this.EnsureComponent<WorldAnchor>();
```

Comment:
I do agree that Get/AddComponent should not be called every frame, it is slow. It should probably be done once, maybe in Connected(), and be re-crated wherever a failed serialization is handled.